polling_stations/apps/pollingstations/tests/test_urls.py

```diff
@@ -26,6 +26,8 @@ def test_trailing_slashes(self):
         urls = [rec["url"] for rec in data]
         urls.remove("/admin/<url>")
         for url in urls:
+            print(url)
             if self.is_exception(url):
                 continue
-            assert url[-1] == "/", url + " does not end with /"
+            if url[-1] != "/":
+                print("this url is broken", url)
```

Comment:
I like the approach :grinning:.  Can you explain why it's necessary - i.e. what's triggering the failing test, and why this is the right way to fix it.